build/ts/typography.ts

```diff
@@ -1,13 +1,69 @@
 /**
  * Do not edit directly
- * Generated on Wed, 15 Jun 2022 15:44:11 GMT
+ * Generated on Thu, 16 Jun 2022 19:25:52 GMT
  */
 
-export const GlobalTypographyH1 = {"fontFamily":"Mulish","fontWeight":"700","lineHeight":"40","fontSize":"40","letterSpacing":"0%","paragraphSpacing":"0","textDecoration":"none","textCase":"none"};
-export const GlobalTypographyH2 = {"fontFamily":"Mulish","fontWeight":"700","lineHeight":"40","fontSize":"32","letterSpacing":"0%","paragraphSpacing":"0","textDecoration":"none","textCase":"none"};
-export const GlobalTypographyH3 = {"fontFamily":"Mulish","fontWeight":"700","lineHeight":"32","fontSize":"24","letterSpacing":"0%","paragraphSpacing":"0","textDecoration":"none","textCase":"none"};
-export const GlobalTypographyH4 = {"fontFamily":"Mulish","fontWeight":"700","lineHeight":"24","fontSize":"20","letterSpacing":"0%","paragraphSpacing":"0","textDecoration":"none","textCase":"none"};
-export const GlobalTypographyH5 = {"fontFamily":"Mulish","fontWeight":"700","lineHeight":"24","fontSize":"16","letterSpacing":"0%","paragraphSpacing":"0","textDecoration":"none","textCase":"none"};
-export const GlobalTypographyBody = {"fontFamily":"Mulish","fontWeight":"500","lineHeight":"24","fontSize":"16","letterSpacing":"0%","paragraphSpacing":"0","textDecoration":"none","textCase":"none"};
-export const GlobalTypographyFootnote = {"fontFamily":"Mulish","fontWeight":"500","lineHeight":"16","fontSize":"14","letterSpacing":"0%","paragraphSpacing":"0","textDecoration":"none","textCase":"none"};
-export const GlobalTypographyCaption = {"fontFamily":"Mulish","fontWeight":"500","lineHeight":"16","fontSize":"12","letterSpacing":"0%","paragraphSpacing":"0","textDecoration":"none","textCase":"none"};
\ No newline at end of file
+export const GlobalTypographyH1FontFamily = "Mulish";
```

Comment:
This new format is not an object like previously generated, but matches the `typography.ts` file we have in `hw-admin`. @iryanclarke, is this the format you had in mind?